### PR TITLE
Use the dataDirectory property of balenarc.yml instead of env var

### PR DESCRIPTION
**Pull request by pdcastro**
Motivated by this internal thread:  
https://www.flowdock.com/app/rulemotion/r-beginners/threads/OKORuvuXb01xTm4EIYuXGEEAO1O

And also, it simplifies the masterclass command line examples.

Change-type: patch

### CODE REVIEW

File: README.md
Comment (xginn8) [inline README.md:203]:
```suggestion
it. Note also that some lines above use `>` and others use `>>` as the shell
```

**Review by xginn8 — COMMENTED** [review]
lgtm! feel free to certify when ready

File: README.md
Comment (pdcastro) [reply README.md:203]: My original version (pre review) didn't have the quotes, :-) then I thought it didn't read nicely, so I added the quotes. This is how it looks without the quotes:

> Note also that some lines above use `>` and others use `>>` as the shell

_vs._

> Note also that some lines above use `'>'` and others use `'>>'` as the shell

Perhaps it depends on the background contrast on the user's screen. I like it better with the quotes. :-) There is no danger of the user copying and pasting the quotes because the actual command line examples do not have quotes.

Thanks for the review! 👍

**Comment by pdcastro** [comment]
@balena-ci retest

**Comment by pdcastro** [comment]
`ResinCI/npm/linux-alpine-x86_64-node8  Expected — Waiting for status to be reported`

This Node 8 test is labeled Required, but doesn't get triggered... Trying again.

**Comment by pdcastro** [comment]
I have made one further change. Everywhere where I had deleted `BALENARC_DATA_DIRECTORY=.`, I have added either `cd ~/balenaProduction` or `cd ~/balenaStaging`. If users were on the wrong directory (say the homedir on a newly opened shell tab/window, or on `/tmp`...) and they used `BALENARC_DATA_DIRECTORY=.`, then the CLI would issue an error, and the user would realise that they typed the command on the wrong folder. But this error would no longer happen after I deleted `BALENARC_DATA_DIRECTORY=.`, so the user might not realise why the examples are not working as expected. My remedy for this was to add `cd ~/balenaProduction` or `cd ~/balenaStaging` **as a reminder** to the user of the directory they are expected to be in. I think users will realise that they don't have to repeat the `cd` command if they know that they are already on that directory...

**Review by xginn8 — APPROVED** [review]

**Comment by pdcastro** [comment]
@xginn8, do you happen to have admin permissions over this masterclass repo? If not, I may have to ask for temporary permissions from ops.

There was a Node v8 "Required" test that was pending and blocking the PR from being merged. Tests for Node v10, v12 and v14 were running successfully. I guess balena CI has moved on with the times and no longer tests Node v8 by default. However, another question is why run any Node tests at all, given that this repo has little more than a README.md file. I have added a `.resinci.yml` file in a new commit that instructs balena CI not to run any Node tests at all -- at least this was my intention!

But this isn't enough: I understand we also still need to change this repository settings to no longer require Node tests to pass before PRs can be merged.

Also: thanks again for the reviews Matthew! 👍

**Comment by xginn8** [comment]
@pdcastro unfortunately i do not have admin on this repo, thanks for sorting that out!

**Comment by pdcastro** [comment]
@apostolism has kindly removed the Required tests for us. Reference:  
https://www.flowdock.com/app/rulemotion/r-operations/threads/3ChZahF2Mtzw-iQlMaJYXQxbwSi

**Pull request merged**
